devops/jobs/CheckSesLimits.groovy

```diff
@@ -0,0 +1,101 @@
+/*
+
+    This job expects the following credentials to be defined on the folder
+    tools-edx-jenkins-aws-credentials: file with key/secret in boto config format
+    ses-role-${deployment}_arn: the role to aws sts assume-role
+
+
+*/
+package devops.jobs
+
+import static org.edx.jenkins.dsl.Constants.common_wrappers
+import static org.edx.jenkins.dsl.Constants.common_logrotator
+import static org.edx.jenkins.dsl.DevopsTasks.common_parameters
+import static org.edx.jenkins.dsl.DevopsTasks.common_multiscm
+
+class CheckSesLimits {
+    public static def job = { dslFactory, extraVars ->
+        assert extraVars.containsKey('DEPLOYMENTS') : "Please define DEPLOYMENTS. It should be a list of strings."
+        assert extraVars.containsKey('CRIT_THRESHOLD') : "Please define CRIT_THRESHOLD. It should be a percentage expressed as a float between 0 to 100."
+        assert extraVars.containsKey('WARN_THRESHOLD') : "Please define WARN_THRESHOLD. It should be a percentage expressed as a float between 0 to 100."
+        assert extraVars.containsKey('NOTIFY_ON_FAILURE') : "Please define NOTIFY_ON_FAILURE."
+        assert !(extraVars.get('DEPLOYMENTS') instanceof String) : "Make sure DEPLOYMENTS is a list and not a string"
+        assert (extraVars.get('CRIT_THRESHOLD') instanceof Integer || extraVars.get('CRIT_THRESHOLD') instanceof Double) : "Make sure CRIT_THRESHOLD is a number between 0 and 100"
```

Comment:
Check to see if it's an instance of `Number` instead.

https://docs.oracle.com/javase/7/docs/api/java/lang/Number.html